### ISSUE TITLE
fail: VotingIrregularities.Domain.NotaAggregate.AdaugaNotaHandler

### ISSUE THREAD

**Issue by KrzysztofMadejski** [code4romania/monitorizare-vot#88]
```
mv-api    | info: Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker[1]
mv-api    |       Route matched with {action = "Upload", controller = "Nota"}. Executing action VotingIrregularities.Api.Controllers.Nota.Upload (VotingIrregularities.Api)
mv-api    | info: Microsoft.AspNetCore.Authorization.DefaultAuthorizationService[1]
mv-api    |       Authorization was successful.
mv-api    | info: Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker[1]
mv-api    |       Executing action method VotingIrregularities.Api.Controllers.Nota.Upload (VotingIrregularities.Api) with arguments (, VotingIrregularities.Api.Models.ModelNota) - Validation state: Valid
mv-api    | fail: VotingIrregularities.Domain.NotaAggregate.AdaugaNotaHandler[0]
mv-api    |       An error occurred while updating the entries. See the inner exception for details.
```

But there is no inner exception :/

App has received 404
```
D/OkHttp: <-- 404 Not Found http://10.0.2.2:8080/api/v1/note/ataseaza (585ms)
```

**Comment by KrzysztofMadejski** [comment]
Something happened here: https://github.com/code4romania/monitorizare-vot/blob/master/private-api/app/src/VotingIrregularities.Api/Controllers/Nota.cs#L58-L61

**Comment by KrzysztofMadejski** [comment]
Resolved in #70
